packages/ui/src/components/theme/theme.css

```diff
@@ -0,0 +1,225 @@
+:root {
+  @media (prefers-color-scheme: light) {
+    --agentprism-brand: 0 0 0;
+
+    --agentprism-background: 1 0 0;
```

Comment:
in terms of this repo we could have some custom linter to match this file with tokens.ts

pseudo-code:

```ts
// In test or build step
const cssTokens = parseCssFile('theme.css');
const tsTokens = AGENT_PRISM_TOKENS;

const missing = tsTokens.filter(t => !cssTokens.includes(t));
if (missing.length) throw new Error(`Missing CSS tokens: ${missing}`);
```